co2calculator/parameters.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""__description__"""
+
```

Comment:
If we do not write class docstrings for all classes here, we should atleast include an informal description what these classes in this file do.

Reply:
done